stripe/src/main/java/com/stripe/android/CustomerSession.java

```diff
@@ -691,14 +860,30 @@ static Intent generateErrorIntent(@NonNull StripeException exception) {
 
     public interface CustomerRetrievalListener {
         void onCustomerRetrieved(@NonNull Customer customer);
-        void onError(int errorCode, @Nullable String errorMessage);
+        void onError (int errorCode, @Nullable String errorMessage);
     }
 
     public interface SourceRetrievalListener {
         void onSourceRetrieved(@NonNull Source source);
         void onError(int errorCode, @Nullable String errorMessage);
     }
 
+    abstract class QueryableBroadcastReceiver extends BroadcastReceiver {
```

Comment:
I'd love to sync for a sec on whether we can make out BroadcastReceiver wiring slightly more streamlined. Does queryBroadcastReceivers possibly do what we want?
https://developer.android.com/reference/android/content/pm/PackageManager.html#queryBroadcastReceivers(android.content.Intent, int)